### PR TITLE
Version 5.1.1

### PR DESCRIPTION
**Pull request by snakefoot**

### CODE REVIEW

File: CHANGELOG.MD [hunk 1]
```diff
@@ -4,6 +4,18 @@ See also [releases](https://github.com/NLog/NLog.Web/releases) and [milestones](
 
 Date format: (year/month/day)
 
+### Version 5.1.1 (2022/08/18)
+
+- **NLog.Web.AspNetCore**
+- [#836](https://github.com/NLog/NLog.Web/pull/836) Fixed offset by one in AspNetRequestDurationLayoutRenderer (#836) (@snakefoot)
+- [#839](https://github.com/NLog/NLog.Web/pull/839) Added option LowercaseKeys for ${aspnet-request-querystring} (#839) (@czd890)
+- [#845](https://github.com/NLog/NLog.Web/pull/845) Added layoutrenderer ${aspnet-request-has-posted-body} and ${aspnet-request-stream-id} (#845) (@bakgerman)
+- [#850](https://github.com/NLog/NLog.Web/pull/850) Improved re-Entrancy scope lock for Session Value Layout Render (#850) (@bakgerman)
```
Comment (bakgerman) [inline CHANGELOG.MD:13]: The re-entrant scope lock maybe should also be listed in the NLog.Web section also, since it has an #if for that case.

Comment (snakefoot) [reply CHANGELOG.MD:13]: Maybe we should just skip re-entrant scope lock protection for NLog.Web-classic, since it wasn't needed before?

Comment (bakgerman) [reply CHANGELOG.MD:13]: Oh I did not know that.  we could do that too.

Comment (bakgerman) [reply CHANGELOG.MD:13]: Why does the classic not need it?

Comment (snakefoot) [reply CHANGELOG.MD:13]: Probably because the new ASP.NET Core-engine behaves different from the classic ASP.NET-engine.

Comment (bakgerman) [reply CHANGELOG.MD:13]: I see, thank you for explaining that situation.

Comment (snakefoot) [reply CHANGELOG.MD:13]: Created #853 to skip ReEntrantScopeLock for classic NLog.Web

**Pull request merged**